crates/rome_service/src/settings.rs

```diff
@@ -172,6 +184,35 @@ pub struct LanguageSettings<L: Language> {
     pub globals: Option<IndexSet<String>>,
 }
 
+/// Filesystem settings for the entire workspace
+#[derive(Debug)]
+pub struct FilesSettings {
+    /// File size limit in bytes
+    pub max_size: NonZeroU64,
+}
+
+const DEFAULT_FILE_SIZE_LIMIT: NonZeroU64 =
```

Comment:
```suggestion
/// Limit the size of files to 1.0 MiB
const DEFAULT_FILE_SIZE_LIMIT: NonZeroU64 =
```

Let's keep  the previous comment